packages/api/indexer/processor/hustler.go

```diff
@@ -214,11 +238,44 @@ func (p *HustlerProcessor) ProcessMetadataUpdate(
 		sex = hustler.SexMALE
 		beardID_ := fmt.Sprintf("%s-%s-%d", sex, bodypart.TypeBEARD, new(big.Int).SetBytes(bodyParts[28:29]).Uint64())
 		beardID = &beardID_
+		// beard check
+		_, err = dbClient.BodyPart.Query().Where(bodypart.IDEQ(*beardID)).Only(ctx)
+		if err != nil {
+			*beardID = ""
```

Comment:
Why set it to an empty string here and log after?

Reply:
Because if I dont it throws a nil pointer exception. This happens for female hustlers who shouldnt have beards. Go handles "null" strings in this way with a pointer. I believe we have to have the ability to insert null instead of empty string for the database.